### PR TITLE
Update 2.0 faq

### PR DESCRIPTION
**Pull request by dingjiaweiww**
Update 2.0 faq 
http://10.136.157.23:8090/documentation/docs/zh/api/index_cn.html?reviewVersion=jenkins-doc-review-362

### CODE REVIEW

File: doc/paddle/faq/2.0.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# 2.0 升级常见问题
+
+## 环境变化
+
+##### 问题：paddle 2.0 是否支持 python2，python3.5 ？
+
++ 答复：paddle 2.0依然提供了python2，python35的官方安装包，但在未来的某个版本将会废弃对python2，python35的支持。（python 官方已停止对python2，python35的更新和维护）
```
Comment (TCChenlong) [inline doc/paddle/faq/2.0.md]: 将会废弃对python2，python35的支持 -> 将会不再支持 python2，python3.5；
官方已停止对python2，python35的更新和维护 -> 官方已停止对python2，python3.5的更新和维护

File: doc/paddle/faq/2.0.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+----------
+
+##### 问题：paddle 2.0 是否支持 CUDA9 ？
+
++ 答复：本版本依然提供了CUDA9的官方安装包，但从未来的某个版本起，将会废弃对CUDA9的支持。
```
Comment (TCChenlong) [inline doc/paddle/faq/2.0.md]: 将会废弃对CUDA9的支持。 -> 将会不再支持CUDA9。

File: doc/paddle/faq/2.0.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+##### 问题：paddle 2.0 是否支持 CentOS6.0 ？
+
++ 答复：对于CentOS6.0，本版本仅提供了有限度的支持（仅发布了CentOS6下，Python3.7 CUDA10.2/CUDNN7 和CPU的安装包）。（CentOS6官方已宣布了停止更新和维护）
+
+----------
+
+##### 问题：paddle 2.0 是否支持 AVX指令的x86机器 ？
+
++ 答复：2.0版本依然提供了对不支持AVX指令的x86机器上运行飞桨的支持，但未来的某个版本将会废弃对不支持AVX指令的x86机器的支持。
```
Comment (TCChenlong) [inline doc/paddle/faq/2.0.md:25]: 但未来的某个版本将会废弃对不支持AVX指令的x86机器的支持。 -> 但未来的某个版本将会不再支持不支持AVX指令的x86机器。

File: doc/paddle/faq/2.0.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+3. 删除依赖的第三方库后，如果需要使用这些库，需要重新安装`pip install objgraph`，直接import objgraph可能会报错。。
+
+----------
+
+
+##### 问题：从1.x版本升级为2.0版本，哪些API有变动 ？
+
++ 答复：飞桨框架2.0.0版本推荐用户使用位于paddle根目录下的API，同时在paddle.fluid目录下保留了所有的1.x版本的API，保留对之前版本API体系的支持。
+
+查看API变动的三种方法：
```
Comment (TCChenlong) [inline doc/paddle/faq/2.0.md]: 三种方法？两种方法？

File: doc/paddle/faq/2.0.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+----------
+
+
+##### 问题：2.0 版本中是否有LoDTensor ？
+
++ 答复：2.0 版本没有LoDTensor的概念，统一使用Tensor来表示数据。
+
++ 解决方案：
+
+1. 使用padding/bucketing的方式对数据进行处理后，使用Tensor来表示数据，进行模型训练，具体事例请参考：IMDB 数据集使用BOW网络的文本分类，使用注意力机制的LSTM的机器翻译。
```
Comment (TCChenlong) [inline doc/paddle/faq/2.0.md]: 具体事例请参考 -> 具体示例请参考

File: doc/paddle/faq/2.0.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+##### 问题：1.8开发的静态图代码能在2.0版本中运行吗 ？
+
++ 答复：
+
+所有1.8的静态图模型在2.0版本中都会报错。
+
++ 问题分析：
+
+2.0版本默认开启了动态图模式。即当用户调用import paddle后，此时Paddle已经运行在动态图模式下。基于1.8开发的静态图代码，在2.0版本下直接执行会出错。为此，我们在静态图的一些入口API 中加入了报错检查，例如直接调用fluid.data会遇到如下错误：
```
Comment (TCChenlong) [inline doc/paddle/faq/2.0.md:79]: 即当用户调用 import paddle后 -> 即当你调用 import paddle后

File: doc/paddle/faq/2.0.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+![图片](https://paddlepaddleimage.cdn.bcebos.com/faqimage%2Fbj-fcd837654fa8c5bb15b071ecaad6b92ef632d872.png)
+
++ 解决方案：
+
+1. 原来依赖于默认静态图的代码需要在import paddle后的头部位置加入paddle.enable_static()来开启静态图模式，这样才能正常运行。
```
Comment (TCChenlong) [inline doc/paddle/faq/2.0.md]: 原来依赖于默认静态图的代码 -> 旧版本下(1.8以及1.8之前版本)静态图下的代码

File: doc/paddle/faq/data_cn.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
+# 数据及其加载常见问题
+
+
+##### 问题：如何在训练过程中高效读取数据量很大的数据集？
+
++ 答复：当训练时使用的数据集数据量较大或者预处理逻辑复杂时，如果串行地进行数据读取，数据读取往往会成为训练效率的瓶颈。这种情况下通常需要利用多线程或者多进程的方法异步地进行数据载入，从而提高数据读取和整体训练效率。
+
+paddle中推荐使用 DataLoader，这是一种灵活的异步加载方式。
+
+该API提供了多进程的异步加载支持，用户可通过配置num_workers指定异步加载数据的进程数目从而满足不同规模数据集的读取需求。
```
Comment (TCChenlong) [inline doc/paddle/faq/data_cn.md]: 用户可通过配置 -> 你可通过配置

File: doc/paddle/faq/data_cn.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+----------
+
+
+##### 问题：如何从numpy.array生成一个具有shape和dtype的Tensor?
+
++ 答复：在动态图模式下，可以参考如下示例：
+
+```
+import paddle
```
Comment (TCChenlong) [inline doc/paddle/faq/data_cn.md:46]: import paddle
import numpy as np

File: doc/paddle/faq/params_cn.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
++ 答复：ParamAttr中定义的`regularizer`优先级更高。若ParamAttr中定义了`regularizer`，则忽略Optimizer中的`regularizer`；否则，则使用Optimizer中的`regularizer`。ParamAttr中的学习率默认为1.0，在对参数优化时，最终的学习率等于optimizer的学习率乘以ParamAttr的学习率。
+
+----------
+
+##### 问题：如何导出指定层的权重，如导出最后一层的*weights*和*bias*？
+
++ 答复：
+1. 在静态图中，使用`paddle.static.save_vars`保存指定的vars，然后使用`paddle.static.load_vars`加载对应层的参数值。具体示例请见API文档：[load_vars](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/fluid/io/load_vars_cn.html) 和 [save_vars](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/fluid/io/save_vars_cn.html) 。
+
+2. 在动态图中，使用`paddle.save` API， 并将最后一层的layer.state_dict() 传入至save方法的obj 参数即可， 然后使用`paddle.load` 方法加载对应层的参数值。详细可参考API文档[save](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/framework/io/save_cn.html#save) 和[load](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/framework/io/load_cn.html#load)。
```
Comment (TCChenlong) [inline doc/paddle/faq/params_cn.md]: 因为2.0主推动态图，所以建议动态图在前，静态图在后

File: doc/paddle/faq/params_cn.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+----------
+
+##### 问题：训练过程中如何固定网络和Batch Normalization（BN）？
+
++ 答复：
+
+1. 对于固定BN：设置 `use_global_stats=True`，使用已加载的全局均值和方差：`global mean/variance`，具体内容可查看官网API文档[batch_norm](https://www.paddlepaddle.org.cn/documentation/docs/zh/1.8/api_cn/layers_cn/batch_norm_cn.html)。
```
Comment (TCChenlong) [inline doc/paddle/faq/params_cn.md]: 这个链接需要换到2.0

File: doc/paddle/faq/2.0.md (hunk 1 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/2.0.md]: 好的

File: doc/paddle/faq/2.0.md (hunk 2 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/2.0.md]: 已改

File: doc/paddle/faq/2.0.md (hunk 3 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/2.0.md:25]: 这个不再支持重复了，原句没有语法错误，可以先不改

File: doc/paddle/faq/2.0.md (hunk 4 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/2.0.md]: 已改，谢谢

File: doc/paddle/faq/2.0.md (hunk 5 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/2.0.md]: 已改

File: doc/paddle/faq/2.0.md (hunk 7 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/2.0.md]: 已改

File: doc/paddle/faq/data_cn.md (hunk 8 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/data_cn.md]: 改变了一种说法

File: doc/paddle/faq/data_cn.md (hunk 9 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/data_cn.md:46]: 已加入

File: doc/paddle/faq/params_cn.md (hunk 10 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/params_cn.md]: OK

File: doc/paddle/faq/params_cn.md (hunk 11 above)
Comment (dingjiaweiww) [reply doc/paddle/faq/params_cn.md]: 已改

File: doc/paddle/faq/2.0.md (hunk 3 above)
Comment (TCChenlong) [reply doc/paddle/faq/2.0.md:25]: 废弃对xxx的支持有语病，两个不再支持，第一个是指不再支持xxx的机器，第二个是指不支持AVX指令，所以应该没有重复

File: doc/paddle/faq/train_cn.md [hunk 12]
```diff
@@ -1,94 +1,14 @@
...
-+ 答复：如果fluid里某一层使用`stop_gradient=True`，那么这一层之前的层都会自动 `stop_gradient=True`，梯度不再回传。
++ 答复：如果静态图中某一层使用`stop_gradient=True`，那么这一层之前的层都会自动 `stop_gradient=True`，梯度不再回传。
 
 ----------
 
-##### 问题：请问`fluid.layers.matmul`和`fluid.layers.mul`有什么区别？
+##### 问题：请问`paddle.matmul`和`paddle.multiply`有什么区别？
 
-+ 答复：`matmul`支持broadcast和任意阶相乘。`mul`会把输入都转成两维去做矩阵乘。
++ 答复：`matmul`支持的两个tensor的矩阵乘操作。`mul`是支持两个tensor进行逐元素相乘。
```
Comment (TCChenlong) [inline doc/paddle/faq/train_cn.md:11]: mul -> multiply

**Review by TCChenlong — APPROVED** [review]
LGTM
